policyengine_us/variables/gov/states/ma/doer/liheap/ma_liheap_eligibility.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class ma_liheap_eligibility(Variable):
+    value_type = bool
+    entity = SPMUnit
+    label = "Eligible for the Massachusetts LIHEAP"
+    definition_period = YEAR
+    reference = (
+        "https://www.mass.gov/info-details/learn-about-home-energy-assistance-heap "
+        "https://liheapch.acf.hhs.gov/tables/FY2015/subsidize.htm#MA"
+    )
+    defined_for = StateCode.MA
+
+    def formula(spm_unit, period, parameters):
+        income = add(spm_unit, period, ["irs_gross_income"])
+        threshold = spm_unit("ma_liheap_income_threshold", period)
+        housing_assistance = spm_unit("housing_assistance", period)
+        heat_in_rent = spm_unit("heat_in_rent", period)
```

Comment:
lets use the more general `utilities_included_in_rent` variable here which we already have 

I assume mass is more specific here but we dont need this level of granularity for now - lets keep the repository as lean as possible